components/admin_console/billing/billing_summary/upsell_card.tsx

```diff
@@ -76,19 +78,51 @@ export default function UpsellCard(props: Props) {
     );
     if (props.upsellIsTrial) {
         callToAction = (
-            <CloudStartTrialButton
-                message={
-                    intl.formatMessage(
-                        {
-                            id: props.cta.id,
-                            defaultMessage: props.cta.defaultMessage,
-                        },
-                        props.cta.values,
-                    )
-                }
-                telemetryId={'start_cloud_trial_billing_subscription'}
-                extraClass={ctaClassname}
-            />
+            <>
+                <CloudStartTrialButton
+                    message={
+                        intl.formatMessage(
+                            {
+                                id: props.cta.id,
+                                defaultMessage: props.cta.defaultMessage,
+                            },
+                            props.cta.values,
+                        )
+                    }
+                    telemetryId={'start_cloud_trial_billing_subscription'}
+                    extraClass={ctaClassname}
+                />
+                <p className='disclaimer'>
```

Comment:
Any chance we can reuse this https://github.com/mattermost/mattermost-webapp/blob/9ec5599ba9206bc73237de63e6c38ac8c96fdfc2/components/pricing_modal/start_trial_caution.tsx 
and allow it to take it some custom styling to adapt font size and color?

Reply:
awesome! I will use it! thanks @AGMETEOR